vizro-core/src/vizro/static/css/accordion.css

```diff
@@ -130,6 +131,10 @@ div.page_container .accordion_button:focus {
 }
 
 div.page_container .accordion_button:hover {
-  background-color: var(--background-hover);
+  background-color: var(--state-overlays-selected-hover);
+  color: var(--text-primary);
+}
+
+div.page_container .accordion-button:hover {
   color: var(--text-primary);
 }
```

Comment:
Remove this? You already added it above

Reply:
It is actually different class, accordion-button and accordion_button. Accordion-button refers to accordion titles, and accordion_button refers to page buttons. 
So accordion-button is 'ANALYSIS' button and accordion_button is 'Homepage'  button.